### PR TITLE
Misc. config for dcwiki T5838

### PR DESCRIPTION
**Pull request by Amanda-Catherine**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2394,6 +2394,20 @@
 	'egApprovedRevsAutomaticApprovals' => [
 		'default' => true,
 	],
+	'wgDefaultUserOptions' => [
```
Comment (paladox) [inline LocalSettings.php:2397]: This is already defined in this file.

File: LocalSettings.php [hunk 2]
```diff
@@ -2394,7 +2394,7 @@
 	'egApprovedRevsAutomaticApprovals' => [
 		'default' => true,
 	],
-
+
```
Comment (RhinosF1) [inline LocalSettings.php:2397]: Why change this?

File: LocalSettings.php [hunk 3]
```diff
@@ -2394,7 +2394,6 @@
 	'egApprovedRevsAutomaticApprovals' => [
 		'default' => true,
 	],
-
```
Comment (RhinosF1) [inline LocalSettings.php:2397]: This should be a blank line

**Comment by Amanda-Catherine** [comment]
@RhinosF1 what are you asking me to do? I removed the extra space that you said I shouldn't change, and now you are asking me to put it back?

**Comment by RhinosF1** [comment]
It should be a blank line, not a line with only spaces on.

**Comment by Amanda-Catherine** [comment]
I'm confused... that sounds like the same thing... a line with nothing but a space is blank

**Comment by RhinosF1** [comment]
No it's not. A space is not equal to a blank line. The GitHub diff should show no change to that link.

**Pull request merged**